### ISSUE TITLE
Are all companies compliant with smartworking?

### ISSUE THREAD

**Issue by ptkdev** [italiaremote/awesome-italia-remote#62]
Before accepting a pull request, you should check if the company is [compliant](https://confcommerciolecco.it/lavoro/il-lavoro-agile-o-smart-working/) with smartworking.

I think the repository needs a bot or a questionnaire to tick off various answers like:

- [ ] The company provides a CCNL contract with the words "accordo individuale di smart working" or "lavoro in modalità agile"
- [ ] Does the company provide assets or a money bonus for your work from home?
- [ ] How does communication take place between employees?
- [ ] Are there any annual or monthly transfers or meetings between employees?

etc... etc... then assign a score from 0 to 10 with the relative link to the questionnaire in the `companies/name_of_enterprise.md` folder for more details.

**Comment by mmazzarolo** [comment]
My 2c.
Given that many companies can only hire you as a contractor I think it might be a bit too much to require strict compliance checks on CCNL. 
I think a #60 + maybe a "note" field would be more flexible and easier to maintain.

**Comment by edoardocostantinidev** [comment]
I agree with @mmazzarolo on this issue, plus retrieving for each company CCNL information could be problematic.

**Comment by alessandromr** [comment]
>We can also discuss FullRemote or Mixed... Because is Mixed when? When you go two times per week to the office, one day per week, x days per month? 🤔
@elgorditosalsero I think we can write that down in the readme or find a better name.
I think about FullRemote for companies without an office where every employee (at least in the category) are working remotely. Instead mixed when only some employees work remotely.

**Comment by elgorditosalsero** [comment]
> > We can also discuss FullRemote or Mixed... Because is Mixed when? When you go two times per week to the office, one day per week, x days per month? 🤔
> 
> @elgorditosalsero I think we can write that down in the readme or find a better name. I think about FullRemote for companies without an office where every employee (at least in the category) are working remotely. Instead mixed when only some employees work remotely.

@alessandromr Mhhh, ok.

I think we could move the discussion about those tags to #83, wdyt?

**Comment by alessandromr** [comment]
Sure, I will close this one because I don't see the CCNL stuff as feasible
